services/buildbot/master/twisted_factories.py

```diff
@@ -689,8 +689,12 @@ class TwistedBdistFactory(TwistedBaseFactory):
     def __init__(self, source, uncleanWarnings, arch, pyVersion):
         python = self.python(pyVersion)
         TwistedBaseFactory.__init__(self, python, source, uncleanWarnings)
-        self.addStep(
-            LearnVersion, python=python, package='twisted', workdir='Twisted')
+        learnVersion = LearnVersion(
+            python=python,
+            package='twisted',
+            workdir='Twisted',
+            )
+        self.addStep(learnVersion)
```

Comment:
nope... learnVersion was failing due to a strange buildbot plugin error... so I have start by decomposing it and planning to put some breakpoints... this code can be reverted, as the LearnVersion was fixed in another part